src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -2,26 +2,75 @@
 
 #define FLAGS ACTOR_FLAG_4
 
-// Horizontal Plane parameters
-
-#define PLANE_Y_MIN -50.0f
-#define PLANE_Y_MAX 200.0f
-
-#define PLANE_HALFWIDTH 100.0f
-#define PLANE_HALFWIDTH_2 200.0f
+// Horizontal holls parameters (`ENHOLL_H_*`)
```

Comment:
```suggestion
/* Horizontal holls parameters (`ENHOLL_H_*`) */
```
Just a styling suggestion, don't have to take it. But with this big block of comments/defines, this might help pop the sections a bit. Same suggestion to
```c
/* Vertical holls parameters (`ENHOLL_V_*`) */
```
or maybe some equal signs in there idk:
```c
/* === Vertical holls parameters (`ENHOLL_V_*`) === */
```

Reply:
I made a multiline block comment out of the h/v "headers", lmk how it looks

(I avoid things like `======` breaks because it's ~~hard~~ impossible to keep consistent across the repo)